test/server/lib/DocApi.ts

```diff
@@ -4846,11 +4922,13 @@ function testDocApi() {
           await check({url: "http://example.com"}, 403, "Provided url is forbidden");  // not https
 
           // changing table without changing the ready column should reset the latter
-          await check({tableId: 'Table2'}, 200, '', expectedFields => expectedFields.isReadyColumn = null);
-
+          await check({tableId: 'Table2'}, 200, '', expectedFields => {
+            expectedFields.isReadyColumn = null;
+            expectedFields.columnIds = "";
+          });
 
           await check({tableId: 'Santa'}, 404, `Table not found "Santa"`);
-          await check({tableId: 'Table2', isReadyColumn: 'Foo'}, 200);
+          await check({tableId: 'Table2', isReadyColumn: 'Foo', columnIds: ""}, 200);
```

Comment:
What happens if we pass `columnIds: ";"` or `columnIds: "A;"`?

Reply:
Good question

Reply:
:heavy_check_mark: I have add a filter to ignore the empty string and a test to ensure it is working